### PR TITLE
UC-73 Fixes for FileUpload

### PR DESCRIPTION
**Pull request by pawel-napieracz**
Exposed useUploadFile and some of FileUpload's interfaces
Fixed parsing maxSize
Fixed filename alignment
Extend useUploadFile hook
Changed error message handler

### CODE REVIEW

File: src/hooks/useUploadFile.test.ts [hunk 1]
```diff
@@ -183,16 +181,13 @@ describe("should return data according to the parameters", () => {
...
-      useUploadFile(files5, { ...requestInfo, networkErrorText: error })
-    );
+    const { result } = renderHook(() => useUploadFile(files5, { ...requestInfo }));
     const [[, readystatechange]] = mock.addEventListener.mock.calls;
     await waitFor(() => readystatechange());
 
     expect(mock.addEventListener).toHaveBeenCalled();
     const file = result.current.updatedFiles[0];
-    expect(file.error).toEqual(error);
+    // expect(file.error).toEqual(error);
```
Comment (MichalGorskiOnegini) [inline src/hooks/useUploadFile.test.ts:190]: ? :D

File: src/index.ts [hunk 2]
```diff
@@ -150,7 +159,8 @@ export type { Props as TextareaWrapperProps } from "./components/Form/Wrapper/Te
 export { SelectWrapper } from "./components/Form/Wrapper/SelectWrapper/SelectWrapper";
 export type { Props as SelectWrapperProps } from "./components/Form/Wrapper/SelectWrapper/SelectWrapper";
 export { FileUpload } from "./components/Form/FileUpload/FileUpload";
```
Comment (MichalGorskiOnegini) [inline src/index.ts:161]: While at it, can you break the line here and introduce a comment e.g /* FileUpload */ ?

**Review by MichalGorskiOnegini — CHANGES_REQUESTED** [review]

File: src/hooks/useUploadFile.test.ts (hunk 1 above)
Comment (pawel-napieracz) [reply src/hooks/useUploadFile.test.ts]: I fixed it with second push ;)

File: src/index.ts (hunk 2 above)
Comment (pawel-napieracz) [reply src/index.ts:163]: added

**Review by MichalGorskiOnegini — APPROVED** [review]

**Review by MichalGorskiOnegini — APPROVED** [review]

**Pull request merged**